js/framework/AnalyticsConstants.js

```diff
@@ -271,7 +271,8 @@ if (!OO.Analytics.EVENTS)
     /**
      * @public
      * @event OO.Analytics.EVENTS#VIDEO_ERROR
-     * @description This message is sent when a video error occurs.
+     * @description (NOTE: will deprecate and be replaced by OO.Analytics.EVENTS.ERROR#VIDEO_PLAYBACK)
```

Comment:
Use the [DEPRECATED] tag from my other comment and keep the note about what it's replaced by.